### PR TITLE
chore: added screenshot section to PR template

### PR DESCRIPTION
**Pull request by DestinedCodes**
## Description:
I added a screenshot section where contributors would attach a screenshot showing how the new feature or bug fix works

## Changes Made:
Added few lines of Markdown for the screenshot section to the existing PR_TEMPLATE.md file

## Related Issue:
No issue was opened for this PR

## Testing Done:
This PR can only be tested after it's been merged with the default branch (main)

### CODE REVIEW

**Review by zxenonx — COMMENTED** [review]
Let's use the PR template I provided.
Thanks again.

**Comment by DestinedCodes** [comment]
The PR template has been replaced with the one provided.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,12 +1,36 @@
...
-<!--- Replace 1234 with the Issue number youre fixing, if it isnt linked to an issue then remive this section-->
-
-## Testing Done:
-Describe the tests added to ensure the new implementation works as expected, including any test cases, inputs, and results.
+<!--- Provide a general summary of your changes in the Title above -->
+​
+## Description
+<!--- Describe your changes in detail -->
+​
+## Related Issue
```
Comment (zxenonx) [inline .github/PULL_REQUEST_TEMPLATE.md:6]:
```suggestion
## Related Issue (Link to linear ticket)
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,12 +1,36 @@
...
+​
+## Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
+​
+## How Has This Been Tested?
+<!--- Please describe in detail how you tested your changes. -->
+<!--- Include details of your testing environment, and the tests you ran to -->
+<!--- see how your change affects other areas of the code, etc. -->
+​
+## Screenshots (if appropriate):
```
Comment (zxenonx) [inline .github/PULL_REQUEST_TEMPLATE.md:20]:
```suggestion
## Screenshots (if appropriate - Postman, etc):
```

**Review by zxenonx — CHANGES_REQUESTED** [review]
Thanks @DestinedCodes 
I left some comments.

**Comment by zxenonx** [comment]
@DestinedCodes i committed my suggestions
You can merge now.

**Comment by DestinedCodes** [comment]
I cannot merge. Branch is protected (whoever opens a PR can't merge it and it cannot be bypassed). Please do merge @zxenonx

**Review by zxenonx — APPROVED** [review]

**Review by Cofucan — APPROVED** [review]

**Pull request merged**